### PR TITLE
Network  Overview mockup

### PR DESCRIPTION
**Pull request by esarafianou**
Depends on #160

### CODE REVIEW

File: etc/spec/mockups/background-shades.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<polygon fill="#FAF4F1" points="1009.2,195.488 836.207,223.384 1078.436,358.408 1150.333,327.318 "/>
+<polygon fill="#F8EFEC" points="1720.388,80.808 1009.2,195.488 1150.333,327.318 "/>
+<polygon fill="#F8EFEC" points="2000,35.559 2000,0 1909.568,0 1719.888,81.308 "/>
+<polygon fill="#FAF4F1" points="1077.936,358.408 0,824.326 0,1200 2000,1200 2000,1121.915 1338.528,503.809 "/>
+<polygon fill="#F8EFEC" points="1338.278,503.809 2000,1121.915 2000,872.671 "/>
+<polygon fill="#F8EFEC" points="835.707,223.384 0,358.062 0,824.326 1077.686,358.408 "/>
+<polygon fill="#F6EAE7" points="1720.388,81.308 1909.568,0 798.848,0 1009.2,196.238 "/>
+<polygon fill="#F6EAE7" points="433.673,0 0,0 0,358.062 835.707,223.884 "/>
+<polygon fill="#F8EFEC" points="1009.2,195.988 798.848,0 433.673,0 836.207,224.134 "/>
+</svg>
```
Comment (dimkarakostas) [inline etc/spec/mockups/background-shades.svg]: change color to something better

File: etc/spec/mockups/networkOverview.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    <div id="mainpage">
+        <div class="line">
+            <ul id="completedattacks">
+                <li class="completed">
+                    <img src="green_pc.png" alt="completed attack">
+                    <ul>
+                        <li> 192.168.1.3</li>
+                        <li> Macintoch Apple</li>
+                        <li> Gmail </li>
+                        <li> Attack completed</li>
```
Comment (dimkarakostas) [inline etc/spec/mockups/networkOverview.html]: attack status in bold and first in list

File: etc/spec/mockups/networkOverview.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                    <ul>
+                        <li> 192.168.1.5</li>
+                        <li>TurboX</li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <footer>
+        <p>Wifi scan image created by icon54 from Noun Project</p>
```
Comment (dimkarakostas) [inline etc/spec/mockups/networkOverview.html]: change logo and remove mention

File: etc/spec/mockups/networkOverview.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        <title>Network Overview</title>
+</head>
+
+<body>
+    <header>
+        <input type="image" src="logo.png" name="logo" onclick='location.href="networkOverview.html"'/>
+        <h1>Network Overview</h1>
+    </header>
+    <div class="button">
+        <input type="image" src="wifiScan.png" name="wifi scan"/>
```
Comment (dimkarakostas) [inline etc/spec/mockups/networkOverview.html]: in window mode scan logo is not placed correctly

**Comment by dimkarakostas** [comment]
Maybe add a time field, eg 01:32:47 under the status bar for running/paused systems, in order to know how long the attack has been running for each system.

**Comment by dimkarakostas** [comment]
Perhaps divide the three different machine states (completed/attacking/not started) some way. Check Bootstrap's [tabs](http://getbootstrap.com/2.3.2/components.html#navs) if you find anything interesting.

File: etc/spec/mockups/networkOverview.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    <div id="mainpage">
+        <ul class="line">
+            <li class="completed">
+                <img src="green_pc.png" alt="completed attack">
+                <ul>
+                    <li class="state">Completed</li>
+                    <li>192.168.1.3</li>
+                    <li>Macintoch Apple</li>
+                    <li>gmail</li>
+                    <li><progress class="progress" value="100" max="100">0%</progress></li>
```
Comment (dimkarakostas) [inline etc/spec/mockups/networkOverview.html]: "progress progress-striped active" is more beautiful I think

File: etc/spec/mockups/networkOverview.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+                </ul>
+            </li>
+            <li class="completed">
+                <img src="green_pc.png" alt="completed attack"</li>
+                <ul>
+                    <li class="state">Completed</li>
+                    <li>192.168.1.4</li>
+                    <li>Dell</li>
+                    <li>Facebook</li>
+                    <li><progress class="progress" value="100" max="100">0%</progress></li>
```
Comment (dimkarakostas) [inline etc/spec/mockups/networkOverview.html]: consider changing the color of the progress bar depending on the state, eg "progress-success" for completed, "progress-info" for attacking etc

File: etc/spec/mockups/networkOverview.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+    <link rel="stylesheet" href="bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Network Overview</title>
+</head>
+
+<body>
+    <header>
+        <input type="image" src="logo.png" name="logo" onclick='location.href="networkoverview.html"'/>
+        <h1>Network Overview</h1>
+    </header>
```
Comment (dimkarakostas) [inline etc/spec/mockups/networkOverview.html]: Use [fixed topbar](http://getbootstrap.com/2.3.2/components.html#navbar) for logo, page header and perhaps network scanner.

**Comment by dionyziz** [comment]
screenshot

**Comment by dionyziz** [comment]
See #170 comment about deletion confirmation modal styling.

**Comment by dionyziz** [comment]
PC icons and labels/progress bars underneath are not horizontally aligned.

**Comment by dionyziz** [comment]
Would you consider using a different icon for a laptop and a mobile phone for clarity? We can use heuristics to detect them.

**Comment by esarafianou** [comment]
networkoverview1

**Comment by esarafianou** [comment]
networkoverview2

**Comment by esarafianou** [comment]
networkoverview3

**Comment by esarafianou** [comment]
deleteattack

**Comment by esarafianou** [comment]
screenshot from 2016-08-04 13-09-00

**Comment by dionyziz** [comment]
Fix build

**Comment by esarafianou** [comment]
screenshot from 2016-08-04 13-09-05

**Comment by esarafianou** [comment]
screenshot from 2016-08-04 13-09-12

**Comment by esarafianou** [comment]
screenshot from 2016-08-04 13-09-19

**Comment by esarafianou** [comment]
screenshot from 2016-08-04 13-09-26

**Comment by dionyziz** [comment]
LGTM

**Pull request merged**